arcgis-ios-sdk-samples/Layers/Display subtype feature layer/README.metadata.json

```
@@ -0,0 +1,26 @@
+{
+  "category": "Features",
+  "description": "Display a composite layer of all the subtype values in a feature class.",
+  "ignore": false,
+  "images": [
+    "Display-subtype-feature-laer.png"
+  ],
+  "keywords": [
+    "asset group",
+    "feature layer",
+    "labeling",
+    "sublayer",
+    "subtype",
+    "symbology",
+    "utility network",
+    "visible scale range"
+  ],
+  "relevant_apis": [
+    "AGSLabelDefinition",
+    "AGSServiceFeatureTable",
+    "AGSSubtypeFeatureLayer",
+    "AGSSubtypeSublayer"
+  ],
+  "snippets": [],
+  "title": "Display subtype feature layer"
+}
```

```suggestion
}

```
I'd like to have a trailing blank line for any text file.